### PR TITLE
fixed /posts/ endpoint and improved profile background

### PR DESCRIPTION
**Pull request by amalik2**

### CODE REVIEW

File: core/posts/serializers.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 from core.authors.serializers import get_summary
 from core.hostUtil import get_host_url
 
-PAGE_SIZE = 50
+PAGE_SIZE = 5
```
Comment (chaitali) [inline core/posts/serializers.py]: Not sure, but https://github.com/i-5ive/CMPUT404-project-socialdistribution/blob/9aff5c342ee85c223f368778b1aa2cf3a415c6b7/example-article.json#L77, has the size as 50.

Comment (amalik2) [reply core/posts/serializers.py]: Idk either, but the lines below it say to choose a comment size like 5 so yea

Comment (chaitali) [reply core/posts/serializers.py]: Yeah, we'll know sooner or later... this should be fine

**Comment by cmackenzie1** [comment]
@amalik2 it is still complaining about eslint on CredentialForm.jsx

**Pull request merged**